configuration.yaml

```
@@ -459,11 +566,38 @@
     - config/crd
   repository: https://github.com/cloudnative-pg/plugin-barman-cloud
   versionPrefix: v
+- entries:
+    - clusternet-agent
+    - clusternet-hub
+  kind: helm
+  name: clusternet
+  repository: https://clusternet.github.io/charts
+- entries:
+    - clusterpedia
+  kind: helm
+  name: clusterpedia
+  repository: https://clusterpedia-io.github.io/clusterpedia-helm/
+  valuesFiles:
+    - version: 0.0.0
+      valuesFile: |
+        installCRDs: true
```

```suggestion
- kind: git
  kustomizationPaths:
    - kustomize/crds
  name: clusterpedia
  repository: https://github.com/clusterpedia-io/clusterpedia
  searchPaths:
    - deploy/crds
  versionPrefix: v
```

I was unable to make this helm chart produce crds